### PR TITLE
Migrate start, stats, stop to Ginkgo

### PR DESCRIPTION
**Pull request by baude**
Migrate the start, stats, and stop integration tests to the
Ginkgo style.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: test/e2e/stats_test.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	It("podman stats only output cids", func() {
+		session := podmanTest.Podman([]string{"run", "-d", "-t", ALPINE, "sleep", "99"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		session = podmanTest.Podman([]string{"stats", "--no-stream", "--format", "\"{{.Container}}\""})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+	})
+
+	It("podman stats with json output", func() {
```
Comment (mheon) [inline test/e2e/stats_test.go]: At some point we should try validating the JSON output's contents, not just that it decodes

**Comment by mheon** [comment]
LGTM. There's a lot of room for improvement in the tests now that we have a proper programming language to write them in, but there's plenty of time for that after BATS is purged.

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**